src/Chat/Client/Actions/PinOrUnpinMessageAction.php

```diff
@@ -14,6 +14,12 @@ public function processResponse($response, int $attempt): int
             return self::SUCCESS;
         }
 
+        if($response === 'Only a room-owner can pin messages') {
```

Comment:
// TODO @DaveRandom Suggests checking the response code, or to see if there's any other way Jeeves can identify a failure other than direct string comparison.